app/services/ajax.ts

```
@@ -0,0 +1,324 @@
+          ...this.defaultHeaders,
+          ...options.headers,
+        }
+      : options.headers || {};
+
+    const fetchOptions: RequestInit = {
+      ...options,
+      headers,
+      body: options.data ? JSON.stringify(options.data) : undefined,
+    };
+
+    delete (fetchOptions as RequestOptions).namespace;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)